sync_csv.sh

```diff
@@ -96,6 +96,9 @@ else
   echo -e "${rtable} already exists, skipping."
 fi
 
-cd $BASE
+echo -e "Attempting to generate reports..."
+cd $HOME/code
+. generate_reports.sh $table
```

Comment:
Do we need to run this twice, once for HAR and once for CSV?

Reply:
Yes. The script only generates reports if all 4 requisite tables exist, so running it after each pipeline ensures that it will succeed regardless of which one finishes last.

If either sync script is run manually (eg to overwrite a bug, or fix a stuck pipeline) it also ensures that the reports are updated accordingly.